pub_sub/csharp/order-processor/order-processor.csproj

```diff
@@ -1,13 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
+    <Nullable>disable</Nullable>

```

Comment:
It would be good to leave nullability enabled if possible. If there are problems I can help sort them out :)